### DISCUSSION TITLE
[FR] extend craftnet API to get all owned licenses

### DISCUSSION THREAD

**Discussion by lukas-jansen** [craftcms/console#61] [Craft Console]
Now it is only possible to query for all licenses created for your own plugins, it would be nice to be able to get all the licenses inside your account/organization

**Comment by brandonkelly** [comment]
When we launched Craft Console, all Plugin Store listings were reassigned to organizations. So the [Plugin Licenses API](https://docs.api.craftcms.com/plugin-licenses.html) was updated to return plugin licenses for all Plugin Store listings owned by any organizations that you are a member of.

Are you saying that’s not working for you?

Or are you asking for a completely new API for returning the same data you’re seeing in Console under “Licenses → Craft CMS” and “Licenses → Plugins” (licenses owned by you/your organizations, for third party plugins)?

**Reply by lukas-jansen** [reply]
Hi Brandon,
I was indeed talking about getting all the third party plugins linked to the organizations.

**Reply by brandonkelly** [reply]
Gotcha, thanks for the clarification.

**Reply by ghost** [reply]
Hi @brandonkelly , I was also after this feature was available to retrieve all the licenses owned by my organisation, not just thrid party plugins but also craft cms licenses.

Asking for an API for returning the same data I'm seeing in Console under “Licenses → "Craft CMS” as well as “Licenses → "Plugins” (any licenses owned by my organisations, for cms + third party plugins)
